src/middleware/normalize.rs

```diff
@@ -10,20 +10,29 @@ use regex::Regex;
 use crate::service::{ServiceRequest, ServiceResponse};
 use crate::Error;
 
-#[derive(Default, Clone, Copy)]
+/// To be used when constructing `NormalizePath` to define it's behavior.
+#[derive(Clone, Copy)]
+pub enum TrailingSlash {
+    /// Always add a trailing slash to the end of the path.
```

Comment:
Make a note that all routes defined must end in trailing slash for them to to be accessible.